.github/workflows/linux_simd.yml

```diff
@@ -198,15 +198,59 @@ jobs:
         sde -spr -- python -c "import numpy; numpy.show_config()" &&
         sde -spr -- python -m pytest $NUMPY_SITE/numpy/_core/tests/test_simd*
```

Comment:
```suggestion
```
There is no longer a need for this test, it would be better to move to the SPR job.

Reply:
sure. Moved it to `inte_sde_spr` job.

Reply:
No, what I meant is to append `$NUMPY_SITE/numpy/_core/tests/test_simd*` to the action `- name: linalg/ufunc/umath tests on Intel SPR` to speed up the test.

Reply:
oh ok. But this probably fine too, right? It's the same set of tests just running in a different job.

Reply:
and the SIMD tests take only a min, when compared to the umath tests take like 12 min.

Reply:
Ok, lets put it in and try to optimize it later, thank you.